src/components/two-factor-authentication/TwoFactorAuthenticationSetUp.vue

```diff
@@ -5,14 +5,15 @@
   import LayoutCard from '@/components/cards/LayoutCard.vue';
   import SpshTooltip from '@/components/admin/SpshTooltip.vue';
   import SoftwareTokenWorkflow from './SoftwareTokenWorkflow.vue';
+  import HardwareTokenWorkflow from './HardwareTokenWorkflow.vue';
   import { useI18n, type Composer } from 'vue-i18n';
 
   const { mdAndDown }: { mdAndDown: Ref<boolean> } = useDisplay();
   const { t }: Composer = useI18n({ useScope: 'global' });
   const selectedOption: Ref<'software' | 'hardware'> = ref('software');
   const personStore: PersonStore = usePersonStore();
 
-  const requestedSoftwareToken: Ref<boolean> = ref(false);
+  const requestingToken: Ref<boolean> = ref(false);
```

Comment:
Ich verstehe die Umbenennung von requested in requesting nicht. Alternativ sowas wie tokenIsRequested?

Reply:
changed